mcp/src/apis/getUsers.ts

```diff
@@ -50,24 +61,37 @@ export const getUsersFactory: ApiFactory<
       ...(includeTimezone ? ['tz'] : []),
     ];
 
-    const res = await pgPool.query<User>(
-      /* sql */ `
-SELECT ${fields.join(', ')}
-  FROM slack.user 
-  WHERE NOT deleted 
+    const args: string[] = []
+    
+    let query = `
+    SELECT ${fields.join(', ')}
+    FROM slack.user 
+    WHERE NOT deleted 
     AND NOT is_bot
-    AND (
-      $1::text IS NULL
-      OR id ILIKE $1
-      OR real_name_normalized ILIKE $1
-      OR display_name_normalized ILIKE $1
-      OR real_name ILIKE $1
-      OR display_name ILIKE $1
```

Comment:
I think removing these will have a negative effect on names containing certain characters. Unless `ILIKE` handles more than capitalization? The purpose of the normalized names is to match `Cécile` with `Cecile`